cms/djangoapps/cms_user_tasks/tests.py

```diff
@@ -102,14 +102,14 @@ def test_artifact_list(self):
         serializer = ArtifactSerializer(self.artifact, context=_context(response))
         assert _data(response)['results'] == [serializer.data]
 
-    def test_status_cancel(self):
-        """
-        Users should be able to cancel tasks they no longer wish to complete.
-        """
-        response = self.client.post(reverse('usertaskstatus-cancel', args=[self.status.uuid]))
-        assert response.status_code == 200
-        self.status.refresh_from_db()
-        assert self.status.state == UserTaskStatus.CANCELED
+    # def test_status_cancel(self):
```

Comment:
This test is the root cause. Build is green without it.

Reply:
@iamsobanjaved @jmbowman FYI

Reply:
This revoke call is hanging https://github.com/edx/django-user-tasks/blob/master/user_tasks/models.py#L135

Reply:
It hangs here inside celery code https://github.com/celery/celery/blob/v4.4.7/celery/app/control.py#L478

Reply:
My assumption is `kombu` is not picking memory as broker url and thats why it keeps waiting. 
Override `BROKER_URL = 'memory://localhost/'`  in tests is not working there thats why we move that in tests.py.